tools/train_svm_low_shot.py

```diff
@@ -6,23 +6,19 @@
 from argparse import Namespace
 from typing import Any, List
 
+from hydra.experimental import compose, initialize_config_module
```

Comment:
@akainth015 , @grace-omotoso , do we know what triggered this change? it doesn't seem aligned with the linter to me.

Reply:
Seems correct to me, moved from line 13 to up here, probably because hydra is considered 3rd party

Reply:
gotcha, thank you for clarifying. This currently conflicts with the linter we have at facebook which expects the alphabetical ordering. Is it possible to address this?

Reply:
Hm, does that mean alphabetical within the same block of imports? We could remove the `no_lines_before = FIRSTPARTY` directive, perhaps that would solve the issue? Although it would result in a lot of changes to existing files.

Reply:
Hi @akainth015 I am new to the vissl team at FB. This PR looks great -- it will be a huge help to us!! 

I think this may have been caused by removing the `known_third_party above`?

Reply:
Hi @iseessel, nice to meet you! Glad that this will be useful

We could add `extra_scripts` to the `known_third_party` option; that would address the issue. Semantically, is that acceptable?